index.php

```diff
@@ -1,10 +1,30 @@
 <?php 
-$db = new PDO('mysql:host=db;dbname=boardgames', 'root', 'password');
-$db->setAttribute(PDO::ATTR_ERRMODE, PDO::ERRMODE_EXCEPTION);
-$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
-$query = $db ->prepare("SELECT `name`, `description`, `playercount`, `difficulty` FROM `boardgames`");
-$query->execute();
-$data = $query->fetchAll();
-echo '<pre>';
-var_dump($data);
-echo '</pre>';
+require('functions.php');
+
+?> 
+
+<html lang="en-GB"> 
+	<head>
+		<link rel="stylesheet" type="text/css" href="style.css">
+		<title>Theboardgamecollection</title>
+	</head>
+	<body>
+        <header>
+            <h1> Boardgame Collection </h1>
+        </header>
+		<div>
+        	<section class = "game">			
```

Comment:
semantically speaking I would call these "card" or "game-card"...my personal preference though so up to you